CHANGELOG.md

```diff
@@ -97,11 +99,59 @@ The following libraries are now considered stable and complete:
 * [Secret Manager API](google/cloud/secretmanager/README.md) [[quickstart]](google/cloud/secretmanager/README.md)
 * [Cloud Tasks API](google/cloud/tasks/README.md) [[quickstart]](google/cloud/tasks/README.md)
 
+## [BigQuery](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/README.md)
+
+* feat(generator): add options support to generated clients ([#7683](https://github.com/googleapis/google-cloud-cpp/pull/7683))
+
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+* feat(pubsublite): stream factories for PSL bidi streams ([#7671](https://github.com/googleapis/google-cloud-cpp/pull/7671))
```

Comment:
This can be deleted. (Not just because it is a pubsublite change in the bigtable section. The PR was an internal only change)

Also, do we want to add pubsublite to this list now that it has handwritten changes? https://github.com/googleapis/google-cloud-cpp/blob/b289fad7602a2960057ec3c74ed245911eba0746/release/changes.sh#L75-L82

Reply:
In the past, we only document changes in libraries once they are GA.

Reply:
Removed. I can add pubsublite to the changes.sh script if we want. If I added it now it would include:

```markdown
### [Pub/Sub Lite](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/pubsublite/README.md)

* feat(pubsublite): stream factories for PSL bidi streams ([#7671](https://github.com/googleapis/google-cloud-cpp/pull/7671))
* feat(generator): add options support to generated clients ([#7683](https://github.com/googleapis/google-cloud-cpp/pull/7683))
```

If that's useful to include, I can add it.